### ISSUE TITLE
The initial learning rate

### ISSUE THREAD

**Issue by dkobak** [wangxin0716/ssl#7]
The initial learning rate appears to be set at 0.6 (base learning rate 0.3, scaled by batch_size / 256 = 2), but actually it is not. You are using `LambdaLR` where `lr_lambda` specifies the multiplicative factor, and your `lr_lambda` value starts with 0.6. So in fact the initial learning rate ends up being 0.6 * 0.6 = 0.36, corresponding to the base learning 0.18.

Relevant parts of the code:

```Python
learning_rate: 0.6 # initial lr = 0.3 * batch_size / 256

def get_lr(step, total_steps, lr_max, lr_min):
    """Compute learning rate according to cosine annealing schedule."""
    return lr_min + (lr_max - lr_min) * 0.5 * (1 + np.cos(step / total_steps * np.pi))

optimizer = torch.optim.SGD(
    model.parameters(),
    args.learning_rate,
    momentum=args.momentum,
    weight_decay=args.weight_decay,
    nesterov=True
)

 scheduler = LambdaLR(
     optimizer,
     lr_lambda=lambda step: get_lr(  # pylint: disable=g-long-lambda
         step,
         args.epochs * len(train_loader),
         args.learning_rate,  # lr_lambda computes multiplicative factor
         1e-3)
)
```
